### DISCUSSION TITLE
Databases not recognized with --enabledb option on singularity command line

### DISCUSSION THREAD

**Discussion by wsnyder4** [brainvisa/brainvisa.github.io#81] [Q&A] [answered]
Hi BrainVISA experts,

I have been running BrainVISA from the command line using the axon-runprocess command with the --enabledb flag to run the Import from FreeSurfer to Morphologist process. I am running BrainVISA with Singularity on a Linux-based cluster.

Each time the script gets to the T1MRI Import process, I keep getting the error:
Mandatory argument <em>output_database</em> of None has no value

I'm running a singularity command that binds to a temporary directory (BVSIF points to the BrainVISA .sif file and BVIMPORT is the FreeSurfer Import .bvproc file I created)
singularity run -B ${TMP_PATH}:${TMP_PATH} scripts/${BVSIF} axon-runprocess --enable-db scripts/${BVIMPORT}

I have set up databases with both BrainVISA and FreeSurfer ontologies, but these are not getting automatically picked up by the process. Is there any way to force it to recognize my chosen databases? Or is there something I should be doing differently with singularity?
Occasionally when I create a database it will only show this change in the ~/.brainvisa/options.minf file (in my home folder), and sometimes it will show in the /casa/home/.brainvisa/options.minf file (in brainvisa's home folder).

Any help here will be greatly appreciated!

Best,
Will

**Comment by c-langlet** [comment, marked as answer]
Hi, 

I am assuming that you followed the steps indicated here (https://brainvisa.info/web/download.html#install-with-singularity) to install Brainvisa, so that you created a directory brainvisa-5.0.0 in your $HOME and installed brainvisa using the singularity command provided in the doc. If you did another installation let us know.

Make sure your database is referenced in ~/brainvisa-5.0.0/home/.brainvisa/options.minf

Can you try :  

    ~/brainvisa-5.0.0/bin/bv axon-runprocess --enabledb scripts/${BVIMPORT}

Clément.

**Reply by wsnyder4** [reply]
That worked! I found that I had multiple copies of BrainVISA and that the older one's bv command was getting recognized over the newer one's bv command in the PATH variable.
